packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -162,11 +175,23 @@ def _delete_ec2_volume(name, timeout=300):
     def _delete_volume(volume):
         volume.delete()  # Raises ClientError if the volume is still attached.
 
-    volumes = list(boto3.session.Session(
-        aws_access_key_id=AWS_ACCESS_KEY_ID,
-        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
-        region_name=AWS_REGION,
-    ).resource('ec2').volumes.filter(Filters=[{'Name': 'tag:Name', 'Values': [name]}]))
+    def _get_current_aws_region():
+        try:
+            return requests.get('http://169.254.169.254/latest/meta-data/placement/availability-zone').text.strip()[:-1]
+        except requests.RequestException as ex:
+            logging.warning("Can't get AWS region from instance metadata: {}".format(ex))
+            return None
+
+    # Remove AWS environment variables to force boto to use IAM credentials.
+    with _remove_env_vars('AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'):
+        volumes = list(boto3.session.Session(
+            # We assume we're running these tests from a cluster node, so we
+            # can assume the region for the instance on which we're running is
+            # the same region in which any volumes were created. If we can't
+            # find the current region, we default to reading it from the
+            # environment.
+            region_name=(_get_current_aws_region() or os.getenv('AWS_REGION')),
```

Comment:
Why do we need this many fallbacks? 

Boto3 has a built in environment variable configuration usage: https://boto3.readthedocs.io/en/latest/guide/configuration.html#environment-variable-configuration

Can we just use that rather than making a new key here?